### PR TITLE
De-duplication of content

### PR DESCRIPTION
**Pull request by Jaanki**

### CODE REVIEW

**Comment by mangelajo** [comment]
@Jaanki you need to rebase and remove the first patch

git rebase -i HEAD~3 
then push it again :)

**Comment by Jaanki** [comment]
@mangelajo done

File: src/content/quickstart/openshift/create_clusters.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+### Create and deploy cluster A
+
+In this step you will deploy cluster A, with the default IP CIDRs
```
Comment (mangelajo) [inline src/content/quickstart/openshift/create_clusters.md:3]: Remember the PR was talking about taking the common bits from here: https://github.com/submariner-io/submariner-website/commit/9fa6eee454c719daa01d0cbff63302779da5d7d4

Can you update this text (and other common texts from .. src/content/quickstart/openshiftgn/_index.md ) it has more review and fixes (at least the phrasings, ... 

Be aware of the cmdline and IP ranges which are different.

File: src/content/quickstart/openshift/ready_clusters.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+### Make your clusters ready for submariner
+
+Submariner gateway nodes need to be able to accept traffic over ports 4500/UDP and 500/UDP
+when using IPSEC. In addition we use port 4800/UDP to encapsulate traffic from the worker nodes
+to the gateway nodes and ensuring that Pod IP addresses are preserved.
```
Comment (mangelajo) [inline src/content/quickstart/openshift/ready_clusters.md:5]: same here ensuring->ensure

Use the bits from here: src/content/quickstart/openshiftgn/_index.md

**Comment by Jaanki** [comment]
@mangelajo as it is change of content, I thought to do it as a separate patch.

**Pull request merged**